source/learn/tutorials/wordpress.rst

```diff
@@ -32,27 +32,57 @@ If everything is working so far and you have all of your login details and passw
 
-That's it - you have a website! We told you it was easy.
+6. Set up automatic updates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. danger::
-    While installing WordPress is quite simple, you must follow the instructions below to ensure that you take the right precautions and protect your website. WordPress is a regular target of hackers and instances of WordPress on the SRCF are regularly compromised in various ways, leading to private data leaks, tampered files, and website irregularities. Most of these risks can be minimized by following all steps in this tutorial. It is important you regularly check up on the status of your installation and keep an eye on any vulnerabilities in the plugins you use!
+WordPress will likely refuse to do automatic updates, instead requesting FTP credentials – this is because it assumes it can't write its own files on the server. To make this work you can add the following few lines to your ``wp-config``.php, near the bottom but above the line that says "stop editing":
+
+.. code-block:: apacheconf
 
-6. Customising WordPress
+  /**
+  * Force WordPress to use direct filesystem access so that upgrades work properly.
+  * https://core.trac.wordpress.org/ticket/10205
+  * https://codex.wordpress.org/Editing_wp-config.php
+  */
+  define('FS_METHOD', 'direct');
+  define('FS_CHMOD_DIR', (02775 & ~ umask()));
+  define('FS_CHMOD_FILE', (0664 & ~ umask()));
+
+If configured correctly, the updates page should include text similar to *Future security updates will be applied automatically*.
+
+If you have a low-traffic or private site, WordPress' cron service (which handles updates and other background tasks) may not run regularly enough. You can invoke ``wp-cron.php`` manually using cron or systemd timers – see ``crontab`` or ``wordpress-cron.timer`` in ``/public/societies/sample``.
+
+.. warning::
+  If you are installing WordPress for a group or society then you will need to ensure that the permissions on files in your group account directory are group-writable (and therefore writable by the user that WordPress will run as). WordPress will not check in advance – if some of your files are writable and some are not, you will end up with a half-upgraded WordPress. You can ensure the permissions are correct by running ``srcf-soc-permfix socname``.
+
+7. Harden your install
+~~~~~~~~~~~~~~~~~~~~~~
+
+WordPress is a regular hackers' target and instances of WordPress on the SRCF are regularly compromised in various ways, leading to private data leaks, tampered files, and website irregularities. Most of these risks can be minimized by following the hardening steps below.
+
+1. Ensure your ``wp-config.php`` is not world-readable, as that will contain your database credentials. You can set permissions in your FTP client or in the console, eg. ``chmod 0660`` to get ``-rw-rw----``
```

Comment:
We've had a few group account admins note that they can't chmod the file WordPress creates, as it's owned by the society account.  Whilst we're here, it would be good to note the use of society sudo for fixing those cases.

Reply:
Fixed in #20.